### PR TITLE
Redact ACCTVERIFY amounts in transaction exports

### PR DESCRIPTION
**Pull request by cjdenio**

### CODE REVIEW

File: app/services/export_service/csv.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ def row(ct)
         [
           ct.date,
           ct.local_hcb_code.memo,
-          ct.amount_cents,
+          @public_only && ct.likely_account_verification_related? ? 0 : ct.amount_cents,
```
Comment (garyhtou) [inline app/services/export_service/csv.rb]: would a NULL be prefered?

**Comment by garyhtou** [comment]
good catch!

File: app/services/export_service/csv.rb (hunk 1 above)
Comment (cjdenio) [reply app/services/export_service/csv.rb]: I think 0 is better so it doesn't mess up scripts that expect a number. This is what we do in the transparency API

Comment (garyhtou) [reply app/services/export_service/csv.rb]: cool!

**Pull request merged**